app/models/host.rb

```diff
@@ -116,7 +116,7 @@ class Host < ApplicationRecord
   virtual_delegate :cpu_total_cores, :cpu_cores_per_socket, :to => :hardware, :allow_nil => true, :default => 0, :type => :integer
   virtual_delegate :num_cpu,     :to => "hardware.cpu_sockets",        :allow_nil => true, :default => 0, :type => :integer
   virtual_delegate :total_vcpus, :to => "hardware.cpu_total_cores",    :allow_nil => true, :default => 0, :type => :integer
-  virtual_delegate :ram_size,    :to => "hardware.memory_mb",          :allow_nil => true, :default => 0, :type => :integer
+  virtual_delegate :ram_size,    :to => "hardware.ram_size_in_bytes",  :allow_nil => true, :default => 0, :type => :integer
```

Comment:
not sure about this one.

leaving alone will work from ruby
not sure what the sql will do.

can you ensure there is a simple spec in host_spec.rb for this one?

just need to make sure it is calling it (and we didn't change the units)

Reply:
i mean i am: https://github.com/ManageIQ/manageiq/pull/20213/files#diff-0b098a46bdc70572954e3f33d98a1c4fR413